ephios/event_management/templates/event_management/fragments/shift_box_big.html

```diff
@@ -41,11 +41,8 @@ <h5 class="card-title">
                                                      <p>{{ error.message }}</p>
                                                  {% endfor %}"
                                                 {% endif %}>
-                                            <button type="submit" class="btn btn-outline-success"
-                                                    name="signup_choice" value="sign_up"
-                                                    {% if not can_sign_up %}
-                                                    style="pointer-events: none;" disabled
-                                                    {% endif %}>
+                                            <button type="submit" name="signup_choice" value="sign_up"
+                                                    class="btn btn-outline-success {% if not can_user_decline %}btn-disabled" disabled {% else %}"{% endif %}>
```

Comment:
should be "can_sign_up" here?

Also can we rename the other one to `can_decline` (remove `user`)

Reply:
(Rename includes the template tags right?)